bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -184,9 +166,131 @@
 		</config-description>
 	</thing-type>
 
-	<channel-type id="alert-level">
+	<channel-type id="vent">
+		<item-type>Number</item-type>
+		<label>Etat Vent</label>
+		<description>Niveau d'alerte aux évènements éoliens</description>
+		<category>vent</category>
```

Comment:
I believe our convention is to use an uppercase for the first letter.
Will it work without mentioning the icon set ? The server will search against all providers ?
To be sure to match the right icon, this will be better to add the icon set, hoping there is no restriction on the value inside this XML tag.

Reply:
There is no apparent restriction in the XML tag. I changed categories to "oh:meteoalerte:xxxx" but I'll need to wait your PR is merged because when I'm queried for an icon, the category holds "meteoalerte:xxxx" and the iconSetId is "classic". I understand it should be "xxxx" for the category and "meteoalerte" for iconSetId.

Reply:
My PR has absolutely no link with that, it only concerns hyphen in icon name.
In which UI are you trying ? If in Main UI, it looks like there is a fix required to be fully compliant with the new syntax. If in Basic UI or Android app, then this would rather be a change required in core framework.

Reply:
Yes, I am testing from Main UI, and the requested iconset is always "classic".

Reply:
Yes, look at the issue, I have captured the network traffic and icon set is not yet handled. I provided a first analysis of the problems based on traffic observation.

In fact, we just have to wait for a proper implementation of icon source / icon set in Main UI.

Then you will be able to test your PR with Main UI.

Reply:
> Yes, look at the issue, I have captured the network traffic and icon set is not yet handled. I provided a first analysis of the problems based on network traffic observation.
> 
> In fact, we just have to wait for a proper implementation of icon source / icon set in Main UI.
> 
> Then you will be able to test your PR with Main UI.

Yes, and it would also be great if Main UI properly display dynamic icons .

Reply:
If you mean the issue openhab/openhab-webui#574 I opened in November 2020, yes it would be great to implement it at the same time, even if less critical.